pom.xml

```diff
@@ -416,7 +416,19 @@
 
 <!-- InMoov2 begin -->
 <!-- skipping fr.inmoov inmoov2 fr.inmoov-inmoov2-null null version/latest -->
-<!-- InMoov2 end -->
+<!-- InMoov2 end -->
+
+<dependency>

```

Comment:
please re-generate the pom.xml to make sure it's still fully in sync with the metadata files.

Reply:
doh .. I did regen it ... I forgot to commit it :(
fixed....